tyk-data-plane/templates/secrets.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     release: {{ .Release.Name }}
 type: Opaque
 stringData:
-  rpcKey: "{{ .Values.global.rpc.key }}"
-  apiKey: "{{  .Values.global.rpc.apiKey }}"
+  rpcKey: "{{ .Values.global.remoteControlPlane.orgId }}"
```

Comment:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/97896463/230353995-5d1021a1-a5ec-484c-a5b1-2ce3dbb71da4.png">
the name of the ENV var is actually rpcKey so I think it offers more clarity to keep it this way for us internally and have the "orgID" one just in the values.yaml file

Reply:
If I understand correctly, secret can be defined by the users too. In that case they will have to use rpcKey in the secret. While in values.yaml, they have to set orgID. It will be inconsistent.

Reply:
Modification has been implemented